public/components/trace_analytics/requests/traces_request_handler.ts

```diff
@@ -389,7 +418,7 @@ export const handlePayloadRequest = (
     .catch((error) => {
       console.error('Error in handlePayloadRequest:', error);
       coreRefs.core?.notifications.toasts.addError(error, {
-        title: `Failed to retrieve payload for trace ID: ${traceId}`,
+        title: `Failed to retrieve payload for trace Id: ${traceId}`,
```

Comment:
We have toast messaged issued here, why aren't we seeing the toast pop up already when the user lands onto a wrong URL on trace view page today?

Reply:
https://github.com/user-attachments/assets/cf041379-2f67-4ba5-ba16-5628602d563c

I believe this is not considered a "error" as it just returns 0. I think our implementation prevents that to stop the toast message spam that was occuring on the landing page before anything was set up by the user.

Reply:
In that case, may be remove this dead code, if it is not reachable

Reply:
Removed.